packages/types/src/scope.ts

```diff
@@ -9,45 +9,65 @@ import { User } from './user';
  */
 export interface Scope {
   /** Add new event processor that will be called after {@link applyToEvent}. */
-  addEventProcessor(callback: EventProcessor): Scope;
+  addEventProcessor(callback: EventProcessor): this;
 
   /**
    * Updates user context information for future events.
-   * @param user User context object to be set in the current context.
+   * @param user User context object to be set in the current context. If null or undefined user will be unset.
```

Comment:
If you pass `''` or `0` it will unset as well, so we should either explicitly check for no arguments at all or say `If passed any falsy value, user will be unset.`